frontend/lib/wallets/Coinbase.ts

```diff
@@ -1,8 +1,9 @@
 import CoinbaseWalletSDK, { CoinbaseWalletProvider } from '@coinbase/wallet-sdk';
 import { ethers } from 'ethers';
-import { getChainIdByNetworkType, getNetworkTypeByChainId } from 'auctions-core/src/network';
+import { getChainIdByNetworkType, getNetworkConfigByType, getNetworkTypeByChainId } from 'auctions-core/src/network';
 import { setSigner } from 'auctions-core/src/signer';
 import { formatToHexWithoutPad } from 'auctions-core/helpers/format';
+import { SITE_TITLE } from '~/nuxt.config';
```

Comment:
Let's not import `nuxt.config` into our files, since it shouldn't be available after build and/or can introduce circular dependencies. Instead, as suggested, pass `SITE_TITLE` to the `env` object and then use `process.env.SITE_TITLE` here

Reply:
Yes